software/aleph.yml

```
@@ -0,0 +1,12 @@
+name: Aleph
+website_url: https://aleph.occrp.org/
+description: "Track people and companies."
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - K8S
+tags:
+  - Miscellaneous
```

Isn't this just search engine?
```suggestion
tags:
  - Search Engines
```